pages/product-selection-opensuse-page.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from '@playwright/test';
+
+export class ProductSelectionPage {
+    readonly page: Page;
+    readonly productSelectionText: Locator;
+    readonly opensuseTumbleweedCheck: Locator;
+    readonly opensuseLeapCheck: Locator;
+    readonly selectButton: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.productSelectionText = page.getByText('Product selection');
+        this.opensuseServerType_Check = page.getByLabel(process.env.PRODUCTNAME);
```

Comment:
it should be opensuseTumbleweedLabel, but could we point to the radiobutton or whatever name is the DOM? it would be more readable.
Please do not forget camel base code style.

Reply:
Updated